src/components/Select/Select.tsx

```diff
@@ -26,25 +28,73 @@ interface Props {
    * Defines the style of the select.
    */
   style?: CSSProperties
+
+  /** Defines the custom error message of the input. */
+  isInvalidFeedback?: string
+  /** Defines the message for valid input. */
+  isValidFeedback?: string
```

Comment:
Since this is not a boolean, I don't think that we should use the `is` prefix. Let's rename this to `validFeedback `